alacritty_terminal/src/term/mod.rs

```diff
@@ -909,7 +912,8 @@ impl<T> Term<T> {
     pub fn new<C>(
         config: &Config<C>,
         size: &SizeInfo,
-        clipboard: Clipboard,
+        title: Option<String>,
```

Comment:
Do we really want to make this optional, just to fallback to a fixed value? What's the intended benefit here?

Reply:
I still wonder what we should do with title. If you look at CI failure you'll notice that we're using it in ConPTY. I decided to use `Option` to simplify tests, however maybe we should put it on a config.

Reply:
I don't see how it answers my question.

The problem is that I don't want to plumb title through the PTY interface only because of Windows. So I was thinking on storing it on a config, which could simplify it a bit.

Reply:
I mean if we need the title, I don't see anything wrong with passing it through to the PTY.

I'd have to actually see what the title does on ConPTY to really judge if we actually need it though.

If https://docs.microsoft.com/en-us/windows/win32/api/processthreadsapi/ns-processthreadsapi-startupinfoa is accurate, this is what it is used for:

> For console processes, this is the title displayed in the title bar if a new console window is created. If NULL, the name of the executable file is used as the window title instead. This parameter must be NULL for GUI or console processes that do not create a new console window.

So it's questionable if this is necessary at all for Alacritty, since we spawn our own Window with its own title. So maybe we don't actually have any problem?

Reply:
I'd go with it for now.

cc @davidhewitt